### ISSUE TITLE
Preview metadata information of search result

### ISSUE THREAD

**Issue by pedro93** [npms-io/npms-www#20]
This is merely a suggestion to a comment thread posted on Reddit about this project.
Adding a preview modal view when hovering over a link to a package on the search result could show a quick summary of the meta information of said package.
Such information could include but is not limited to:
- Downloads in last 30 days
- Amount of GH stars
- Last updated date
- Dependency count
- Dependent count
- Past version count
- Current unstable/RC version
- Direct GH link for quick access to repository

Link to thread in question: [comment](https://m.reddit.com/r/javascript/comments/4nq1ti/npms_a_better_and_open_source_search_for_node/d461tlm)

**Comment by satazor** [comment]
I agree that this would be a good addition. Later on, we will also have a package info page, that slides from the right whenever you click a module. If you click outside it, it will slide out. This way you won't loose the context of your search.

**Comment by darsain** [comment]
Downloads in last 30 days, and link to repository should definitely be visible/accessible without any hover requirement. They are so valuable when discovering packages. I really don't want to hover that proprietary rating stamp for every search result.

Please :)

**Comment by icebob** [comment]
+1 for Github stars. I think it is very important too.

**Comment by satazor** [comment]
I've added a "view package analysis" meanwhile. While this is not the final solution, it's something. Ideally we would open a modal dialog with JSON data highlighted.

**Comment by darsain** [comment]
How can you consider modal windows an ideal solution for anything? They are annoying, a barrier between user and data, and should be implemented only in a situation where you absolutely can't find a different solution.

So, are you sure you can't display a couple numbers without a modal window? :)

**Comment by satazor** [comment]
It's not a couple a numbers but a lot of data to be shown.. showing only the nr of stars and what not is very incomplete. The score is calculated using the whole analysis.

**Comment by satazor** [comment]
Hovering the score could show a limited set of metrics as outlined by the OP (in the tooltip). 

If everyone agrees with it, feel free to make a PR. Adding all this information in the item itself without requiring hovering/click would make items cluttered.

**Comment by darsain** [comment]
> The score is calculated using the whole analysis.

Yeah but I don't care about that score. I don't care about any of those 4 scores that are currently displayed. I simply don't know what they mean. I'd have to learn the algorithms they are calculated with for them to become meaningful. So right now they are just an arbitrary numbers that obfuscate data I'm actually interested in.

Pretty much the most important stuff, and all the numbers I really want to see are:
- version 👍 already displayed, although it could be bigger and drop the parentheses
- installs in last week _(month is too big timeframe for new, and day too low for less popular modules)_
- last update date 👍 already displayed
- _maybe GitHub stars_

And I'd never look at your score again. This is not a huge amount of data that would require a modal window.

**Comment by satazor** [comment]
@darsain those scores are explained in the [about](https://npms.io/about) page. If you don't care to read it, no wonder you don't understand them. These "proprietary" scores give much more value than

> version 👍 already displayed, although it could be bigger and drop the parentheses
> installs in last week
> last update date 👍 already displayed
> maybe GitHub stars

You might agree these are the most important metrics, but I disagree as well as many other community members that I reached before creating npms.io. Members that actually participated in brainstorms and decisions that made npms.io so much popular than other alternatives that only use the metrics you listed.
